### ISSUE TITLE
Flow error: Cannot assign function to match because:

### ISSUE THREAD

**Issue by schumannd** [Khan/simple-markdown#62]
Full error:
```

Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ node_modules/simple-markdown/simple-markdown.js:465:39

Cannot assign function to match because:
 • property 0 is missing in object type [1] but exists in Capture [2] in the return value.
 • ... 1 more error.

     node_modules/simple-markdown/simple-markdown.js
 [2]  78│ ) => ?Capture;
        :
     462│
     463│ // Creates a match function for an inline scoped element from a regex
     464│ var inlineRegex = function(regex /* : RegExp */) {
     465│     var match /* : MatchFunction */ = function(source, state) {
     466│         if (state.inline) {
     467│             return regex.exec(source);
     468│         } else {
     469│             return null;
     470│         }
     471│     };
     472│     match.regex = regex;
     473│     return match;
     474│ };

     /private/tmp/flow/flowlib_102fce2d/core.js
 [1] 288│ type RegExp$matchResult = Array<string> & {index: number, input: string, groups: ?{[name: string]: string}};

```

I am using flow version 0.86

Edit:
there are a few more errors flow 0.86 complains about. If there is interest, I can post their error messages as well.

**Comment by ariabuckles** [comment]
Thanks! I haven't upgraded flow to latest myself; it looks like they improved their regexp match type. I'll look into fixing this and whatever other issues show in flow 0.86+

**Comment by ariabuckles** [comment]
Just published `0.4.3` which should fix this!

**Comment by schumannd** [comment]
Wow thank you so much for the quick response!

I tried it and it works perfectly!
